examples/index.html

```diff
@@ -91,6 +91,37 @@ <h4>Tracks</h4>
                 <p><small>Configure which stream to ingest and store media to. Call update media storage configuration with an empty Stream name to disable this feature.</small></p>
                 <div class="form-group input-group">
                     <input type="text" class="form-control" id="streamName" placeholder="Stream name">
+                    <div class="input-group-append">
```

Comment:
can we have a space in between `CreateStream` and `UpdateMediaStorageConfiguration` button? From the screenshot, it seems like a single button

Reply:
Done 
<img width="1144" alt="image" src="https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-js/assets/14988194/f3306d3e-f578-4902-afbf-cd95fcb4a4cf">